doc/en/components/charts/chart-types-bar.md

```diff
@@ -131,6 +136,8 @@ In this Range Bar Chart example, weather data variance between high and low temp
 
 ## Styling the $PlatformShort$ Bar Chart
 
+In this example, 
+
 <!-- <div class="sample-container loading" style="height: 500px">
```

Comment:
unfinished sentence

Reply:
Added "Annotations can be added to the Bar Chart. The following example demonstrates a percent comparison between popular online retail shopping sites."